### ISSUE TITLE
Allow VOLTTRON management UI to pull data from remote boards

### ISSUE THREAD

**Issue by jhaack** [VOLTTRON/volttron#1028]
Allow Management UI to pull data from remote boards instead of requiring forward agent.

**Comment by acedrew** [comment]
Is this meant to be an on-demand peer platform bus subscription from the VC UI?

**Comment by craig8** [comment]
yeah and it actually can already do it believe it or not...it's basically acting similar to a Prometheus pull.
